scripts/nixops

```diff
@@ -1,262 +1,2 @@
-add_common_deployment_options(subparser)
-
-subparser = add_subparser(subparsers, 'delete-generation', help='remove a previous configuration')
-subparser.set_defaults(op=op_delete_generation)
-subparser.add_argument('generation', type=int, metavar='GENERATION', help='number of the desired configuration (see ‘nixops list-generations’)')
-add_common_deployment_options(subparser)
-
-subparser = add_subparser(subparsers, 'show-console-output', help='print the machine\'s console output on stdout')
-subparser.set_defaults(op=op_show_console_output)
-subparser.add_argument('machine', metavar='MACHINE', help='identifier of the machine')
-add_common_deployment_options(subparser)
-
-subparser = add_subparser(subparsers, 'dump-nix-paths', help='dump Nix paths referenced in deployments')
-subparser.add_argument('--all',  action='store_true', help='dump Nix paths for all deployments')
-subparser.set_defaults(op=op_dump_nix_paths)
-add_common_deployment_options(subparser)
-
-subparser = add_subparser(subparsers, 'export', help='export the state of a deployment')
-subparser.add_argument('--all',  action='store_true', help='export all deployments')
-subparser.set_defaults(op=op_export)
-
-subparser = add_subparser(subparsers, 'import', help='import deployments into the state file')
-subparser.add_argument('--include-keys',  action='store_true', help='import public SSH hosts keys to .ssh/known_hosts')
-subparser.set_defaults(op=op_import)
-
-subparser = add_subparser(subparsers, 'edit', help='open the deployment specification in $EDITOR')
-subparser.set_defaults(op=op_edit)
-
-subparser = add_subparser(subparsers, 'copy-closure', help='copy closure to a target machine')
-subparser.add_argument('machine', help='identifier of the machine')
-subparser.add_argument('storepath', help='store path of the closure to be copied')
-subparser.set_defaults(op=op_copy_closure)
-
-subparser = subparsers.add_parser('list-plugins', help='list the available nixops plugins')
-subparser.set_defaults(op=op_list_plugins)
-subparser.add_argument('--verbose', '-v', action='store_true', help='Provide extra plugin information')
-subparser.add_argument('--debug', action='store_true', help='enable debug output')
-
-parser_plugin_hooks(parser, subparsers)
-
-if os.path.basename(sys.argv[0]) == "charon":
-    sys.stderr.write(nixops.util.ansi_warn("warning: ‘charon’ is now called ‘nixops’") + "\n")
-
-args = parser.parse_args()
-setup_logging(args)
-
-try:
-    nixops.deployment.debug = args.debug
-    args.op(args)
-except deployment.NixEvalError:
-    error("evaluation of the deployment specification failed")
-    sys.exit(1)
-except KeyboardInterrupt:
-    error("interrupted")
-    sys.exit(1)
-except MultipleExceptions as e:
-    error(str(e))
-    if args.debug or args.show_trace or str(e) == "":
-        e.print_all_backtraces()
-    sys.exit(1)
+#!/usr/bin/env bash
+exec python -m nixops "$@"
```

Comment:
Is this just for backwards compatibility at dev time?

Reply:
Exactly.